resources/ts/Components/Integrations/Detail/CredentialsLegacyAuthConsumers.tsx

```diff
@@ -32,7 +32,12 @@ export const CredentialsLegacyAuthConsumers = ({
         <Heading className="font-semibold" level={4}>
           {t("details.credentials.test")}
         </Heading>
-        <CopyText>{legacyTestConsumer?.apiKey}</CopyText>
+        <div className="flex gap-1 max-md:flex-col max-md:items-start">
+          <span className="flex items-center whitespace-nowrap">
+            {t("details.credentials.api_key")}
+          </span>
+          <CopyText>{legacyTestConsumer?.apiKey}</CopyText>
```

Comment:
Is the optional chaining necessary?
In the previous file, you don't do it...
```suggestion
          <CopyText>{legacyTestConsumer?.apiKey}</CopyText>
```